config/locales/en.yml

```
@@ -38,7 +38,7 @@ account_reset.request.info:
 account_reset.request.no_cancel: Cancel
 account_reset.request.title: Account deletion and reset
 account_reset.request.yes_continue: Yes, continue deletion
-account.connected_apps.associated: Connected %{timestamp}
+account.connected_apps.associated_html: Connected %{timestamp}
```

I think we gotta call this `timestamp_html` so the linter doesn't get mad
```suggestion
account.connected_apps.associated_html: Connected %{timestamp_html}
```

---

> I think we gotta call this `timestamp_html` so the linter doesn't get mad

👍 Updated in ac4b9e5aa4